sceptre/sageit/templates/admodeladexplorer-redirector.yaml

```diff
@@ -0,0 +1,150 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: >-
+  Provision an S3 bucket, CF distribution to handle redirects for admodelexplorer and modeladexplorer
+Parameters:
+  StackType:
+    Description: Staging or Prod stack
+    Type: String
+    AllowedValues: ['prod', 'staging', 'test']
```

Comment:
staging or prod but yet allowed values contains test?

Reply:
Removed, I added it for deploying from command-line and forgot to remove...